app/services/arcgis_api/geocoder.rb

```diff
@@ -40,28 +42,41 @@ def suggest(text)
     # @param magic_key [String] a magic key value from a previous call to the #suggest method
     # @return [Array<AddressCandidate>] AddressCandidates
     def find_address_candidates(magic_key)
-      url = "#{root_url}/findAddressCandidates"
+      url = "#{root_url}/servernh/rest/services/GSA/USA/GeocodeServer/findAddressCandidates"
       params = {
         magicKey: magic_key,
         outFields: 'StAddr,City,RegionAbbr,Postal',
         **COMMON_DEFAULT_PARAMETERS,
       }
 
       parse_address_candidates(
-        faraday.get(url, params) do |req|
+        faraday.get(url, params, dynamic_headers) do |req|
           req.options.context = { service_name: 'arcgis_geocoder_find_address_candidates' }
         end.body,
       )
     end
 
+    # Makes a request to retrieve a new token
+    # it expires (1 hour minutes).
```

Comment:
`1 hour minutes` ... one hour?
```suggestion
    # it expires after 1 hour
```